db/schema.rb

```diff
@@ -10,17 +10,27 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-
-ActiveRecord::Schema.define(version: 20200512205827) do
-
-# ActiveRecord::Schema.define(version: 20200512020737) do
-# This was a merge conflict but wasnt sure if deleting it was the best option. Otherwise version above is from mine
-
+ActiveRecord::Schema.define(version: 20200513215451) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "favorites", force: :cascade do |t|
+  create_table "apps", force: :cascade do |t|
+    t.string "name"
+    t.string "address"
+    t.string "city"
+    t.string "state"
+    t.string "zip"
+    t.string "phone_number"
+    t.string "description"
+    t.boolean "approved"
```

Comment:
if you create an app, and set `approved = true` we should be in business. You'll also have to use the join table PetApps to make sure each app is associated with a pet and vise versa (see the playlistsongs from the lesson)

Reply:
Coool coool coool